test/root/integration/octaneConfigConverter.js

```diff
@@ -0,0 +1,46 @@
+/*!
+ * (c) Copyright 2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/* eslint-env mocha */
+
+const fs = require('fs')
+const path = require('path')
+
+exports.convertToRootConfig = function () {
+  const generateRoutesConfig = JSON.parse(fs.readFileSync(path.join(__dirname, '../../../octane.json'), 'utf8'))
```

Comment:
We can and that`s how i made it initially, but then we would need 2 config files for running the tests, one for the root and one for generate routes. Since you need the old structure for the generate routes and the routes need to be generated first before tests for it can be run, i left the old file as the config file for the tests + the CI builds that file already